{{cookiecutter.project_slug}}/tests/CI_docker/build_{{cookiecutter.project_slug}}_testsuite_image.sh

```diff
@@ -1,3 +1,4 @@
+#syntax=docker/dockerfile:experimental
```

Comment:
This line should go in the .docker file and not into this file.

Reply:
done